src/miner.h

```diff
@@ -163,6 +169,12 @@ class BlockAssembler
     static Optional<int64_t> m_last_block_num_txs;
     static Optional<int64_t> m_last_block_weight;
 
+    /* This function wraps addPackageTxs to calculate and return the minimum fee
+     * rate required for a package to currently be included in the highest fee rate
+     * block possible based on mempool transactions.
+     */
+    CFeeRate minTxFeeRate();
```

Comment:
Functions should be capitalized:

```suggestion
    CFeeRate MinTxFeeRate();
```

Reply:
fixed